rubin_sim/maf/metrics/snNSNMetric.py

```diff
@@ -81,17 +87,18 @@ def __init__(
         nexpCol="numExposures",
         vistimeCol="visitTime",
         seeingCol="seeingFwhmEff",
+        noteCol="note",
         season=[-1],
         coadd_night=True,
-        zmin=0.0,
-        zmax=1.2,
+        zmin=0.1,
+        zmax=0.5,
```

Comment:
This should help speed up the metric calculation, if a maximum z limit of 0.5 is acceptable. I take it that this is the maximum value you've found when running the metric on simulations, thus the update? 
Does it need to be different for DD fields?

Reply:
Yes, 0.5 should be the max for WFD runs and this value has been used for the results quoted in the PR (processing time) . For DDFs, one should set: zmin=0.1, zmax=1.0

Reply:
Thank you for the update for DDF fields.